libs/core/src/lib/grid-list/components/grid-list-item/grid-list-item.component.html

```diff
@@ -187,6 +205,8 @@
                 label="Locked"
                 [compact]="true"
                 (click)="_clickOnLocked($event)"
+                (mouseover)="_isActive = false"
```

Comment:
I would move this assignment into ts file, so we could apply ChangeDetector if needed later